src/lib/Makefile

```diff
@@ -101,9 +101,9 @@ endif
 # check if bindir and libdir are consistent.
 # notice that this is not required yet but if we later we want
 # to use $ORIGIN based rpaths this would be needed
-ifneq ($(dir $(bindir)),$(dir $(libdir)))
-  $(error bindir and libdir are not consistent, please set them as subdirectories of the same directory)
-endif
+#ifneq ($(dir $(bindir)),$(dir $(libdir)))
```

Comment:
Is there any specific reason to remove this? Are bindir and libdir in your system not located in the same directory?

Reply:
@aeszter I have seen the comment now:
```
Debian places libraries inside /usr/lib/<arch>, so bindir and libdir
```
Are binary files located in `/usr/bin/` or `/usr/bin/<arch>`?

Reply:
Merged this as well

Reply:
Binaries are in /usr/bin. As far as I understand, the reasoning is that on a given system, installed binaries will always be of the system architecture, but libraries may not (if the system is set up for cross-compiling).